### ISSUE TITLE
Percentage operator

### ISSUE THREAD

**Issue by TonyFred-code** [Captain-Usopp/Calculator#13]
Pressing '25%25' returns '0'... Ought to return 25

**Comment by voidsama73** [comment]
This is happening because he used the percentage operator for modulus. This is a quick fix.

**Comment by TonyFred-code** [comment]
Yeah... It'd odd for that to be a modulo operator given it's a basic calculator.
